### PR TITLE
Fixes #230 | Servo Motor Documentation

### LINKED ISSUE #230: Servo Motor Documentation
- [x]  Component Description - what it can do
- [x]  Add yaml configuration
- [x]  Circuit setup image
- [x]  Schematic diagram
- [x]  Instructions on how to use

Note: Follow the same section order of other components.

### PR DESCRIPTION
**Pull request by adelgadoj3**
Created the basis for the documentation. Need to fill in the remaining sections.

### CODE REVIEW

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+* 1 x T-extension Board
+* 1 x Servo-Motor
+* 3 x Jumper Wires
+* Power source (appropriate voltage, typically 3.3V)
+
+===== Assembly Instructions
+* Connect a jumper wire to 3.3V
+* Connect a jumper wire to GPIO 18
+* Connect a jumper wire to GND
+* Connect the 3.3V jumper wire to the power wire receiver (Color: Red)
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc:34]: mention this as VCC pin

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+* Connect the GND jumper wire to the ground wire receiver (Color: Brown or Black)
+* Connect the GPIO 18 jumper wire to the signal wire receiver (Color: Orange)
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc:36]: mention this as SIG pin

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+
+===== Circuit Diagram
+
+image::servo-motor_CD.png[]
+
+*Schematic Diagram*
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc:43]: i think we have Schematic diagram as a section

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+
+==== Servo Motor
+
+[.text-right]
+
+https://github.com/oss-slu-Pi4Micronaut/edit/develop/pi4micronaut-utils/src/docs/asciidoc/componets/outputComponents/servomotor.adoc[Improve this doc]
+
+===== Overview
+This section provides details of the Servo Motor, including the components, and assembly instructions.
```
Comment (ruthvikm) [inline pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc:19]: There are different types of servo motors. Specify which one we are providing support for. I think the one we are using is a "Micro Servo 9G - SG90" which is a positional servo with the range from 0 to 180 degrees

**Review by ruthvikm — CHANGES_REQUESTED** [review]
Requested few changes

File: pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc (hunk 3 above)
Comment (adelgadoj3) [reply pi4micronaut-utils/src/docs/asciidoc/components/outputComponents/servomotor.adoc:43]: In other components we have it as listed above *schematic diagram* . Would you like it to be its own own section? (i.e. ===== Schematic Diagram)

**Review by ruthvikm — APPROVED** [review]
Looks good to me.

**Comment by jyanev** [comment]
nice (☞ﾟヮﾟ)☞
d=====(￣▽￣*)b

**Pull request merged**